src/installation.rst

```diff
@@ -170,13 +170,15 @@ available too.
 We recommend using a wrapper script named ``cylc`` to activate the correct
 environment before calling the environment's  ``cylc`` command.
 
-.. TODO - update this once the wrapper has been added to cylc-flow package data.
+Cylc comes with a wrapper which can be extracted to a directory in your $PATH
+using
```

Comment:
It doesn't have to be a directory in your `$PATH`, does it?  Just give any target path on the command line.